gallery/validators.py

```diff
@@ -0,0 +1,9 @@
+from django.core.exceptions import ValidationError
+
+def validate_file_size(value):
+    filesize = value.size
+
+    if filesize > 2097152:
+        raise ValidationError("The maximum file size that can be uploaded is 2MB.")
```

Comment:
make it 10MB as limit, but it should compress the image well below 500kb.
For documents, keep 10mb as limit, i dont see compression required for documents.